README.md

```diff
@@ -1,3 +1,42 @@
-# Apache DataFusion Web Site
+# Apache DataFusion Blog Content
 
-Coming soon
+This repository contains the Apache DataFusion blog content.
+
+## Setup for Mac
+
+Based on instructions at https://jekyllrb.com/docs/installation/macos/
+
+```shell
+brew install chruby ruby-install xz
+ruby-install ruby 3.1.3
+```
+
+Note: I did not have a `~/.zshrc` file so had to create one first.
+
+```
+echo "source $(brew --prefix)/opt/chruby/share/chruby/chruby.sh" >> ~/.zshrc
+echo "source $(brew --prefix)/opt/chruby/share/chruby/auto.sh" >> ~/.zshrc
+echo "chruby ruby-3.1.3" >> ~/.zshrc # run 'chruby' to see actual version
+```
+
+Quit and restart terminal.
+
```

Comment:
Eventually it would be nice to have some docker instructions too for those who don't want to install ruby locally

https://github.com/apache/arrow-site?tab=readme-ov-file#using-docker

But we can do that as a follow on PR